runtime/src/common/version.rs

```diff
@@ -1,12 +1,19 @@
+use num_traits::Zero;
```

Comment:
This is in a strange place.

Reply:
I moved it a bit down before the serde stuff.